.travis.yml

```diff
@@ -4,6 +4,7 @@ python:
 install:
     - pip install requests junit_xml beaker-client mock
     - pip install pylint pycodestyle
+    - pip install .
```

Comment:
Shouldn't this be a part of tests instead, i.e. be in the `script` list? And should we perhaps add an uninstall test as well?

Reply:
Sounds reasonable. I'll make those changes.